lib/receptor_controller/client/response_worker.rb

```diff
@@ -47,15 +47,15 @@ def start
         return if started.value
 
         default_messaging_opts # Thread-safe init
-        init_notifications(init_lock, init_wait)
+        # init_notifications(init_lock, init_wait)
```

Comment:
if manageiq-messaging will support both, I'm for keeping this but with condition `if config.ruby-kafka...`

Reply:
It was removed in version 1.0, per https://github.com/ManageIQ/manageiq-messaging/pull/55/files